app/views/membership.html

```diff
@@ -173,14 +173,42 @@
                         class="sr-only">
                         Name
                       </label>
+
+                      <!-- name entry field for everything but service accounts -->
                       <input
+                        ng-if="newBinding.kind !== 'ServiceAccount'"
                         type="text"
                         class="form-control input-name"
                         placeholder="Name"
                         ng-model="newBinding.name">
+
+                      <!-- name entry/picker for service accounts -->
                       <div
                         ng-if="newBinding.kind === 'ServiceAccount'"
-                        class="service-account-namespace hidden-sm hidden-md hidden-lg"
+                        class="service-account-name pad-bottom-sm"
+                        aria-hidden="true">
```

Comment:
Good catch, I need to remove that tag now as I previously had 2 `<ui-select>` for namespace, but now only have 1.